src/main/java/org/alliancegenome/curation_api/services/GafService.java

```diff
@@ -0,0 +1,138 @@
+package org.alliancegenome.curation_api.services;
+
+import jakarta.annotation.PostConstruct;
+import jakarta.enterprise.context.RequestScoped;
+import jakarta.inject.Inject;
+import jakarta.transaction.Transactional;
+import org.alliancegenome.curation_api.auth.AuthenticatedUser;
+import org.alliancegenome.curation_api.dao.GafDAO;
+import org.alliancegenome.curation_api.dao.GeneDAO;
+import org.alliancegenome.curation_api.dao.SpeciesDAO;
+import org.alliancegenome.curation_api.dao.ontology.GoTermDAO;
+import org.alliancegenome.curation_api.model.entities.*;
+import org.alliancegenome.curation_api.model.entities.ontology.GOTerm;
+import org.alliancegenome.curation_api.model.ingest.dto.GeneOntologyAnnotationDTO;
+import org.alliancegenome.curation_api.response.ObjectResponse;
+import org.alliancegenome.curation_api.services.base.BaseEntityCrudService;
+import org.alliancegenome.curation_api.services.validation.DataProviderValidator;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@RequestScoped
+public class GafService extends BaseEntityCrudService<GeneOntologyAnnotation, GafDAO> {
```

Comment:
These should be named consistently so this should be: GeneOntologyAnnotationService